TodoApp/TodoApp.Api/Controllers/ItemListController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
```

Comment:
In a commit like 765bde9d870f677759e642bf8a5cebac53aa46ea, I would actually expect you to remove unused using from all files across your solution (ALT+ENTER → Remove unused directives in file &#9654; Remove unused directives in solution → click)